docs: clarify 'destination' argument (#262)
timescale/pgai: docs/vectorizer-api-reference.md

| Name             | Type                                                   | Default                           | Required | Description                                                                                        |
|------------------|--------------------------------------------------------|-----------------------------------|----------|----------------------------------------------------------------------------------------------------|
| source           | regclass                                               | -                                 | ✔        | The source table that embeddings are generated for.                                                |

> | destination      | name                                                   | -                                 | ✖        | A name for the table the embeddings will be or are stored in. If defined, *view_name* argument takes this value, and the value of the *target_table* argument is set as `<destination>_store`.                                     |

Reviewer: I find it confusing that we reference other arguments to explain how this works.

```suggestion
| destination      | name                                                   | -                                 | ✖        | Sets the name of the embedding table and view. Use this to avoid naming conflicts when configuring additional vectorizers for a source table. The view is named `<destination>`, and the embedding table is named `<destination>_store`.                                     |
```

Author: I felt we should clarify that, normally, you either define `destination` or `target_table` and `view_name`. How could we add this without naming those args?

Reviewer 2: Set the name of table embeddings are stored in, and the view \<whatever it does>. The view is named `<destination>`, the embedding table is named `<destination>_store`. You set `destination` to avoid naming conflicts when you configure additional vectorizers for a source table.

Author: I ended up doing a mix of both of your suggestions. Please check 🙏

| chunking         | [Chunking configuration](#chunking-configuration)      | -                                 | ✔        | Set the way to split text data, using functions like `ai.chunking_character_text_splitter()`.      |
| indexing         | [Indexing configuration](#indexing-configuration)      | `ai.indexing_default()`           | ✖        | Specify how to index the embeddings. For example, `ai.indexing_diskann()` or `ai.indexing_hnsw()`. |
| formatting       | [Formatting configuration](#formatting-configuration)  | `ai.formatting_python_template()` | ✖        | Define the data format before embedding, using `ai.formatting_python_template()`.                  |
| scheduling       | [Scheduling configuration](#scheduling-configuration)  | `ai.scheduling_default()`         | ✖        | Set how often to run the vectorizer. For example, `ai.scheduling_timescaledb()`.                   |
| processing       | [Processing configuration](#processing-configuration ) | `ai.processing_default()`         | ✖        | Configure the way to process the embeddings.                                                       |

> | target_schema    | name                                                   | -                                 | ✖        | Specify the schema where the embeddings will be stored. This argument takes preference over *destination*.                                     |

Reviewer: ```suggestion
| target_schema    | name                                                   | -                                 | ✖        | Specify the schema where the embeddings will be stored. This argument takes precedence over *destination*.                                     |
```

> | target_schema    | name                                                   | -                                 | ✖        | Specify the schema where the embeddings will be stored. This argument takes precedence over *destination*.                                     |

Reviewer 2: ```suggestion
| target_schema    | name                                                   | -                                 | ✖        | Specify the schema where the embeddings will be stored. This argument takes precedence over `destination`.                                     |
```

| target_table     | name                                                   | -                                 | ✖        | Specify name of the table where the embeddings will be stored.                                     |
| view_schema      | name                                                   | -                                 | ✖        | Specify the schema where the view is created.                                                      |

> | view_name        | name                                                   | -                                 | ✖        | Specify the name of the view to be created. This argument takes preference over *destination*.                                     |

Reviewer: ```suggestion
| view_name        | name                                                   | -                                 | ✖        | Specify the name of the view to be created. This argument takes precedence over *destination*.                                     |
```

> | view_name        | name                                                   | -                                 | ✖        | Specify the name of the view to be created. This argument takes precedence over *destination*.                                     |

Reviewer 2: ```suggestion
| view_name        | name                                                   | -                                 | ✖        | Specify the name of the view to be created. This argument takes precedence over `destination`.                                     |
```
